samples/SqlServerCacheSample/project.json

```diff
@@ -0,0 +1,19 @@
+{
+    "commands": {
+        "install-sqlservercache": "Microsoft.Framework.Caching.SqlServer",
```

Comment:
yes, this creates tables as of now...i didn't do much polish here as there is other idea that came up yesterday about creating tables at run time...